src/utils/playSound.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable import/prefer-default-export */
+import { debounce } from './lodash-replacements';
+
+// Play a given sound
+export const playSound = ({
+  src,
+  loop = false,
+  volume = 1,
+  debounceInterval = 1000,
```

Comment:
The delay between the trigger and the sound feels too long, and sometimes doesn't fire at all, for quicker consecutive sound interactions (e.g., adding nodes from form). Do you think this could be related to this debounce mechanism? If so, maybe there is a way to edit this to still prevent rapid fire while also making the interactions feel responsive.

Reply:
trimming trailing silence in [0340356](https://github.com/complexdatacollective/Interviewer/pull/1241/commits/03403566c125ccb6cc22be2ec551bfbf21e46b98) seems to improve this